x-pack/plugins/canvas/canvas_plugin_src/uis/models/math.js

```diff
@@ -5,16 +5,24 @@
  */
 
 import { get } from 'lodash';
+import { i18n } from '@kbn/i18n';
 import { getState, getValue } from '../../../public/lib/resolved_arg';
 
 export const math = () => ({
   name: 'math',
-  displayName: 'Measure',
+  displayName: i18n.translate('xpack.canvas.uis.models.mathDisplayName', {
```

Comment:
please add .math namespace for all ids in this file. and check other files in models folder, add them their own namespace also